### DISCUSSION TITLE
checking cloudflare recaptcha detects selenium

### DISCUSSION THREAD

**Discussion by krzysztofkub** [seleniumbase/SeleniumBase#3301] [General]
Hi,

I use seleniumbase version:` 4.33.1 ` on windows

This is my code:

```
import sys
from seleniumbase import SB
from PIL import Image
import io
import requests
import time
import subprocess
import winsound


def vote():
    with SB(uc=True, test=True, incognito=True) as sb:
        url = "https://www.xtremetop100.com/in.php?site=1132376426"
        sb.uc_open_with_reconnect(url)
        resolve_captcha(sb)


def resolve_captcha(sb):
    # resolve image captcha manually
    time.sleep(10)
   
    button_element = 'input[name="ticki"]'
    sb.reconnect(0.1)
    sb.uc_click(button_element, reconnect_time=4)
    sb.uc_gui_click_captcha()
    sb.wait_for_text_not_visible("Checking", timeout=12)
    time.sleep(10)


if __name__ == "__main__":
            vote()
```

So when you enter https://www.xtremetop100.com/in.php?site=1132227942, you need to put code from image manually. Then script clicks button to send form and page with cloudflare recaptcha shows up. When I click it manually it works fine. But when I use sb.uc_gui_click_captcha() looks like server detects that it's not human

Could you help me please? 

Thanks

**Comment by krzysztofkub** [comment]
I've changed my code a little bit (updated snippet) to be more similar to the examples

**Comment by egor-ryashin** [comment]
I wonder if it's a specific browser (Chrome) or all other are affected too?
